.circleci/workflows.yml

```diff
@@ -451,6 +463,9 @@ workflows:
       - x86_64-linux-dist:
           requires:
             - x86_64-linux-build
+      - x86_64-linux-dist-tools:
+          requires:
+            - x86_64-linux-build
```

Comment:
This job is not a dependency of any other job, while all jobs must (indirectly) be a dependency of the `finish-build` job. Otherwise the `finish-build` job will run, marking the commit as "releasable". The most appropriate job to add this as a dependency is `x86_64-linux-self-test`.